lib/org/chromium/webidl/LocalGitRunner.js

```diff
@@ -5,18 +5,14 @@
 
 foam.CLASS({
   package: 'org.chromium.webidl',
-  name: 'LocalGitIDLFileDAO',
-  extends: 'foam.dao.ProxyDAO',
+  name: 'LocalGitRunner',
+  extends: 'foam.box.Runnable',
```

Comment:
I intended on extending `foam.box.Runnable` then wiring up an additional box (i.e. an alternate box property) where URLs can be sent out. The default output box will be required (this is used by all the paths and it streams the IDL files) and the alternate box (where the final list of URLs is sent) will be optional.

I can easily change it to not extend `Runnable` if that make it look a little cleaner. Let me know what you think about the options